runtime/doc/news.txt

```diff
@@ -181,6 +182,9 @@ EVENTS
 
 • |CmdlineLeavePre| triggered before preparing to leave the command line.
 • New `append` paremeter for |ui-messages| `msg_show` event.
+• New `msg-id` and `extra-info` paremeter for |ui-messages| `msg_show` event.
```

Comment:
This doesn't match api-ui-events.txt

Reply:
Names changed a bit from review. Forgot to update it here. Will do.